python/interpret_community/common/model_wrapper.py

```diff
@@ -205,6 +205,8 @@ def predict(self, dataset):
         if is_sequential or isinstance(self._model, WrappedPytorchModel):
             return self._model.predict_classes(dataset).flatten()
         preds = self._model.predict(dataset)
+        if isinstance(preds, pd.DataFrame):
+            preds = preds.values.ravel()
```

Comment:
Yes ravel was needed in case of regression problems. the .values on pandas dataframe returns a 2-d numpy array with shape like (len, ). So ravel is to make that a linear array.